checkipa

```diff
@@ -245,8 +263,10 @@ class IntegrityCheck(object):
         elif self.ipa_info['enterprise'] and not self.ipa_info['adhoc']:
```

Comment:
this should not be "elif", it should just be "if". An enterprise profile can have an expiration date